src/backend/binaryen/utils.ts

```diff
@@ -1185,6 +1185,15 @@ export namespace FunctionalFuncs {
                     ),
                 );
             }
+            case ts.SyntaxKind.CaretToken: {
+                return convertTypeToF64(
+                    module,
```

Comment:
Can you also add this operation in `operateF32F32`, `operateI32I32` and `operateI64I64` ?

Reply:
Resolved. Thanks for advise